testing/pypy3/const_stdio.patch

```diff
@@ -0,0 +1,18 @@
+diff -r b30914a448be rpython/rlib/rfile.py
```

Comment:
Why is this patch needed (please add patch header with description)? Have you sent it also to upstream, or is it specific for Alpine?